umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-json-schema.json

```diff
@@ -0,0 +1,604 @@
+          "type": "string",
+          "minLength": 0,
+          "maxLength": 4000
+        }
+      },
+      "required": ["Identifier"]
+    },
+    "IndexRangesType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The index ranges consist of a LatRange and a LonRange.",
+      "properties": {
+        "LatRange": {
+          "description": "The LatRange consists of an index range for latitude.",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 2,
+          "maxItems": 2
+        },
+        "LonRange": {
+          "description": "The LonRange consists of an index range for longitude.",
+          "type": "array",
+          "items": {
+            "type": "number"
+          },
+          "minItems": 2,
+          "maxItems": 2
+        }
+      },
+      "required": ["LatRange", "LonRange"]
+    },
+    "MeasurementIdentifierType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The elements of this section allow authors to provide community sourced words or phrases to further describe the variable data.",
+      "properties": {
+        "MeasurementContextMedium": {
+          "description": "This element describes the context/medium within which the measurement was made.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        },
+        "MeasurementContextMediumURI": {
+          "description": "This element contains the URI for the context/medium.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 4000
+        },
+        "MeasurementObject": {
+          "description": "This element describes the object which was measured.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 1024
+        },
+        "MeasurementObjectURI": {
+          "description": "This element contains the URI for the object which was measured.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 4000
+        },
+        "MeasurementQuantities": {
+          "description": "This element contains the quantity or quantities which was/were measured.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MeasurementQuantityType"
+          },
+          "minItems": 0
+        }
+      },
+      "required": ["MeasurementContextMedium", "MeasurementObject"]
+    },
+    "MeasurementQuantityType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The elements of this section apply to a measurement name. The measurement name is structured according to the form defined by Scott Peckham. This is: <object>_<quantity>.",
```

Comment:
I found a person with this name at the University of Co, in Bolder, but it feels like a really strange definition of a format. Should a regex exist to enforce any of this?

Reply:
unless we know of all the measurement values that will fit in reg-ex, we can't enforce this.  We don't know what characters are valid.

Reply:
I think I'm more bothered by the fact that the definition of the field is a person.